streamlit_ace/__init__.py

```diff
@@ -27,8 +26,8 @@ def st_ace(
     show_gutter=True,
     show_print_margin=False,
     readonly=False,
-    annotations=[],
-    markers=[],
+    annotations=None,
+    markers=None,
```

Comment:
This is a Python gotcha that I'd never remember if my editor didn't auto-lint it for me: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Nice catch! I sometime directly use lists as default argument when I know I won't mutate them in my function's body, but it's a good practice not to use them in case I forget this gotcha or change the function later on, indeed 😄